packages/cli/src/ceramic-daemon.ts

```diff
@@ -143,22 +143,21 @@ class CeramicDaemon {
   }
 
   registerAPIPaths (app: core.Express, gateway: boolean): void {
-    app.get(toApiPath('/records/ceramic/:docid'), this.records.bind(this))
-    app.get(toApiPath('/show/ceramic/:docid'), this.show.bind(this))
-    app.get(toApiPath('/state/ceramic/:docid'), this.state.bind(this))
-    app.get(toApiPath('/pin/ls/ceramic/:docid'), this.listPinned.bind(this))
-    app.get(toApiPath('/pin/ls'), this.listPinned.bind(this))
-    app.get(toApiPath('/supported_chains'), this.getSupportedChains.bind(this))
-    app.post(toApiPath('/create'), this.createDocFromGenesis.bind(this))
+    app.get(toApiPath('/records/:docid'), this.records.bind(this))
+    app.put(toApiPath('/document'), this.createDocFromGenesis.bind(this))
```

Comment:
This is minor, but my understanding is that HTTP PUT methods are supposed to be idempotent, which createDocFromGenesis won't be for Tile doctypes unless `unique:false` is explicitly provided.  Not sure how much the choice of HTTP method really matters anyway, but figured I'd call that out.

Reply:
Addressed this by changing applyRecord to just use `records`.